### ISSUE TITLE
Width of menuView

### ISSUE THREAD

**Issue by fatgue** [kitasuke/PagingMenuController#302]
Hi,

is it possible to change the width of the menuView? I want add two buttons on the right and left side of the menuView. With "var height: CGFloat {return 44 }" i can change the height but there is not a width option. Any help?

**Comment by kitasuke** [comment]
Does menu view's width corresponds to paging view's width? Or do you want to change width of menu view only?

**Comment by fatgue** [comment]
@kitasuke yes it is correct the menu view's width corresponds to the paging view's width. I will only change the menu's View.
